dotCMS/src/integration-test/java/com/dotcms/enterprise/publishing/remote/DependencyBundlerTest.java

```diff
@@ -565,5 +568,34 @@ public void testGenerateBundleWithFilter_RelationshipsFalse_ChildcontentletChild
         Assert.assertTrue(listOfAssetsWithNewFilter.getRelationships().isEmpty());
     }
 
+    /**
+     * Method to test: {@link DependencyManager#setDependencies()}
+     * Given Scenario: If a page has a Template that was deleted, we should be able to generate the bundle without issues.
+     * ExpectedResult: The bundle should be generated without issues and the page added to the bundle.
+     */
+    @Test
+    public void Test_GenerateBundle_PageWithTemplateDeleted_PageSuccessfullyAddedToBundle() throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5647397)